src/ekomark/benchmark/external/apfel_utils.py

```diff
@@ -72,8 +72,8 @@ def compute_apfel_data(
 
     # Run
     apf_tabs = {}
-    for mu2 in operators["Q2grid"]:
-        apfel.EvolveAPFEL(theory["Q0"], np.sqrt(mu2))
+    for mu in operators["mugrid"]:
```

Comment:
why did you revert?

Reply:
Considering the line below, if `mugrid` is available the current version makes sense (and it avoids the old `Q2` nomenclature)

Reply:
@felixhekhorn I think that the other day we were looking by mistake to an old version of the code. In fact in the current master, in ekomark operator has `mugrid` instead of `Q2grid`